arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -0,0 +1,1670 @@
+          "type": "string"
+        },
+        "timeZone": {
+          "description": "The device Time Zone. For eg: \"Pacific Standard Time\"",
+          "type": "string"
+        },
+        "dnsSettings": {
+          "$ref": "#/definitions/SecondaryDNSSettings",
+          "description": "Secondary DNS Settings of the device."
+        },
+        "networkInterfaceData0Settings": {
+          "$ref": "#/definitions/NetworkInterfaceData0Settings",
+          "description": "Data 0 Network interface card settings."
+        }
+      }
+    },
+    "Device": {
+      "description": "Represents a StorSimple device along with its properties",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseModel"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DeviceProperties",
+          "description": "Collection of properties for the device",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DeviceDetails": {
+      "description": "Additional details having endpoint count and volume container count",
+      "type": "object",
+      "properties": {
+        "endpointCount": {
+          "format": "int32",
+          "description": "Total number of endpoints that are currently on the device ( i.e. number of volumes)",
+          "type": "integer"
+        },
+        "volumeContainerCount": {
+          "format": "int32",
+          "description": "Total number of volume containers on the device",
+          "type": "integer"
+        }
+      }
+    },
+    "DeviceList": {
+      "description": "Collection of Devices",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Device"
+          }
+        }
+      }
+    },
+    "DevicePatch": {
+      "description": "Represents the patch request for the device.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DevicePatchProperties",
+          "description": "Properties of the device patch.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DevicePatchProperties": {
+      "description": "Patch Device properties.",
+      "type": "object",
+      "properties": {
+        "deviceDescription": {
+          "description": "Short description given for the device",
+          "type": "string"
+        }
+      }
+    },
+    "DeviceProperties": {
+      "description": "Encases all the properties of the Device",
+      "required": [
+        "friendlyName",
+        "activationTime",
+        "culture",
+        "deviceDescription",
+        "deviceSoftwareVersion",
+        "deviceConfigurationStatus",
+        "targetIQN",
```

Comment:
One query, shouldn't this camel-case restriction be relaxed for popular abbreviations, like IPv4, IPv6 (and DNS and IQN), even when part of a variable, as the common-form in which they are accepted/known is lost when converted to camelCasing?